class .js

```diff
@@ -0,0 +1,38 @@
+
+////////////////////////////Product 클래스 구현
+
+class product{
+constructor(name, Description, PromiseRejectionEvent, tags=[], images=[]) {
+this.name = name;
+this.Description = Description;
+this.price = price;
+this.tags = tags;
+this.images = images;
+this.favoriteCount = 0;
+}
+}
+favorite(){
+this.favoriteCount++
+console.log(`${this.name}이 찜하기를 할경우 찜하기 수가 ${this.favoriteCount}수가 증가합니다`)
+}
```

Comment:
1. style: 들여쓰기가 안되어있네요. 강사님께 배운 formatter 활용하기(이거 세팅 멘토링 시간에 해볼게요)
2. 매개변수명 수정 필요(기본적으로 변수명은 camelCase쓰기, price대신 PromiseRejectionEvent가 있어서 에러 발생, 중괄호 2개 있어서 아래 favorite이 메서드로 포함되지 않아 문법 에러)
3. 복사본을 넣어야함. 참조 타입은 그대로 넣으면 원본이 변경되면 내부 멤버변수도 변경됨.
```suggestion
  constructor(name, Description, PromiseRejectionEvent, tags=[], images=[]) {
    this.name = name;
    this.Description = Description;
    this.price = price;
    this.tags = [...tags];
    this.images = [...images];
    this.favoriteCount = 0;
  }

  favorite(){
    this.favoriteCount++
    console.log(`${this.name}이 찜하기를 할경우 찜하기 수가 ${this.favoriteCount}수가 증가합니다`)
  }
}
```